be/src/olap/memtable.h

```diff
@@ -57,7 +57,7 @@ class MemTable {
     };
 
     RowCursorComparator _row_comparator;
-    std::unique_ptr<MemTracker> _tracker;
+    std::unique_ptr<MemTracker> _mem_tracker;
```

Comment:
We use 5-level hierarchy of mem trackers:
LoadChannelMgr->LoadChannel->TabletsChannel->DeltaWriter->Memtable.

I think it is OK because:
1. Load process may not has such high concurrency that updating the mem trackers have impact on it.
2. The Memtracker is designed for query process, which is ought to support at least 5 levels: process -> pool -> query -> fragment -> sub fragment. (This is what Memtracker's comment said)

Reply:
OK